helm/v1alpha1/register.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
```

Comment:
any package we create here that is on by default needs to be supported for at least three releases.  I think this is a v1beta1 even though we hope to remove it eventually.

Reply:
renamed to `v1beta1`